DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/storm_control/test_storm_control_unknown_unicast_traffic.py

```diff
@@ -0,0 +1,142 @@
+import pytest
+import asyncio
+
+from dent_os_testbed.utils.test_utils.cleanup_utils import cleanup_kbyte_per_sec_rate_value
+from dent_os_testbed.lib.devlink.devlink_port import DevlinkPort
+from dent_os_testbed.lib.ip.ip_link import IpLink
+from random import randrange
+
+from dent_os_testbed.utils.test_utils.tgen_utils import (
+    tgen_utils_get_dent_devices_with_tgen,
+    tgen_utils_traffic_generator_connect,
+    tgen_utils_dev_groups_from_config,
+    tgen_utils_get_traffic_stats,
+    tgen_utils_setup_streams,
+    tgen_utils_start_traffic,
+    tgen_utils_get_loss
+)
+
+pytestmark = [
+    pytest.mark.suite_functional_storm_control,
+    pytest.mark.asyncio,
+    pytest.mark.usefixtures('cleanup_bridges', 'cleanup_tgen')
+]
+
+
+async def verify_rate_value(dev, name, value, device_host_name):
+    out = await DevlinkPort.show(input_data=[{device_host_name: [
+        {'options': '-j', 'dev': dev, 'name': name}]}], parse_output=True)
+    err_msg = f"Failed to execute the command 'DevlinkPort.show'.\n{out}"
+    assert out[0][device_host_name]['rc'] == 0, err_msg
+    devlink_info = out[0][device_host_name]['parsed_output']
+    kbyte_value = devlink_info['param'][dev][0]['values'][0]['value']
+    assert kbyte_value == value, f"Verify that storm control rate configured is '{value}' kbps.\n"
+
+
+async def devlink_rate_value(dev, name, value, cmode=False, device_host_name=True,
```

Comment:
Isn`t this the same function from the previous file? Move this to storm_control_utils.py